From 791aa7dedc42bf8cbe9dfaeec5a3fd9ce0a055e4 Mon Sep 17 00:00:00 2001
From: WeiYong Bi <bivvy.bi@rock-chips.com>
Date: Wed, 21 Dec 2016 15:20:03 +0800
Subject: [PATCH] arm64: dts: rockchip: support use drm on rk3399 excavator
 linux

1. fixup edp enable gpio.
2. add backlight enable gpio.
3. enable route_edp node.

Change-Id: I4b4792abe3c8eb7dba0c2f5c0d3ddffff74478d1
Signed-off-by: WeiYong Bi <bivvy.bi@rock-chips.com>
---
 .../boot/dts/rockchip/rk3399-sapphire-excavator-linux.dts  | 7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-sapphire-excavator-linux.dts b/arch/arm64/boot/dts/rockchip/rk3399-sapphire-excavator-linux.dts
index 86afc876f2b9..64b37120e263 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-sapphire-excavator-linux.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-sapphire-excavator-linux.dts
@@ -65,7 +65,7 @@
 		compatible = "lg,lp079qx1-sp0v", "panel-simple";
 		backlight = <&backlight>;
 		power-supply = <&vcc3v3_s0>;
-		enable-gpios = <&gpio1 13 GPIO_ACTIVE_HIGH>;
+		enable-gpios = <&gpio4 30 GPIO_ACTIVE_HIGH>;
 		pinctrl-names = "default";
 		pinctrl-0 = <&lcd_panel_reset>;
 
@@ -125,12 +125,17 @@
 
 &backlight {
 	status = "okay";
+	enable-gpios = <&gpio1 13 GPIO_ACTIVE_HIGH>;
 };
 
 &display_subsystem {
 	status = "okay";
 };
 
+&route_edp {
+	status = "okay";
+};
+
 &edp {
 	status = "okay";
 
-- 
2.35.3

